src/main/java/com/synopsys/integration/alert/web/controller/SystemController.java

```diff
@@ -49,4 +54,15 @@ public SystemController(final SystemHandler handler) {
     public ResponseEntity<String> getSystemMessages(@RequestParam(value = "startDate", required = false) final String startDate, @RequestParam(value = "endDate", required = false) final String endDate) {
         return handler.getSystemMessages(startDate, endDate);
     }
+
+    @GetMapping(value = "/system/setup/initial")
```

Comment:
Because we want to add a Setup menu item to display the form the initial configuration needs to be accessible without requiring login.  Therefore the /initial sub path will have permissions to be accessed without login.  When Setup becomes a menu item then /system/setup will be the URL path for that.  Both should be rendered by the same UI Component in the future.  Just wanted to provide some context on the path.